Makefile.toml

```diff
@@ -139,6 +160,57 @@ description = "Generates Elasticsearch client from REST API specs"
 dependencies = ["run-api-generator"]
 run_task = "format"
 
+[tasks.docs]
+description = "Generate Elasticsearch client documentation and opens in browser"
+clear = true
+category = "Elasticsearch"
+command = "cargo"
+args = ["doc", "-p", "elasticsearch", "--no-deps", "--open"]
+
+[tasks.generate-release-notes]
+category = "Elasticsearch"
+description = "Generates release notes for Elasticsearch client"
+# assumes clients-team repo is checked out as a sibling of elasticsearch-rs
+condition = { env_set = ["OLD_VERSION", "NEW_VERSION"], files_exist = [ "${CARGO_MAKE_WORKING_DIRECTORY}/../clients-team/scripts/release-notes-generator/Dockerfile" ] }
+script_runner = "bash"
+script = [
+"""
+pushd ./../clients-team/scripts/release-notes-generator
+docker build --file ./Dockerfile --tag clients-team/release_notes_generator .
+popd
+docker run -v "${CARGO_MAKE_WORKING_DIRECTORY}/.ci/release/config.yml:/usr/src/release_notes_generator/config.yml" --rm clients-team/release_notes_generator -o ${OLD_VERSION} -n ${NEW_VERSION}
+"""
+]
+
+[tasks.generate-release-notes.windows]
```

Comment:
Could the use of `@shell` or Duckscript avoid a platform-specific implementation of this task?

[shell2batch](https://github.com/sagiegurari/shell2batch) doesn't support `pushd` but we can replace it with `cd`

Reply:
++ using `@shell` works